server/routes/enrolled.js

```diff
@@ -1 +1,18 @@
-//TODO
\ No newline at end of file
+
+const router = require('express').Router();
+
+let totalEnrolled = require('../../db/enrolled.model');
+
+const { exampleEnrolledGenerator } = require('../example.data');
+
+router.route('/getEnrolled').get((req, res) => {
+  // using a hack to populate enrolled table
+  totalEnrolled.saveEnrolled(exampleEnrolledGenerator());
+
+
+  totalEnrolled.Enrolled.find().sort({_id: -1}).limit(1)
```

Comment:
So this will return the element with the highest index?

Reply:
Yup or the latest entry added to this table. I probably will refactor this to give highest number or something. Thanks